### DISCUSSION TITLE
Cannot enable color management

### DISCUSSION THREAD

**Discussion by sunlit777** [iina/iina#5082] [Q&A]
Just downloaded IINA, last version, using Mac OS Sonoma, also all updated as of time of writing, apple silicon Macbook Pro - color management is not working. Video picture is off (compared to quicktime and Safari) both with gamma and colors - both on external srgb (rec.709) TV and on built in LCD. Ticking the checkbox "use ICC profile for display" makes things worse than using with this box unchecked. Please advise. If color managment broken, how is this not top priority bug? MOdern Mac users 80-90% have to deal with wide-gamut displays. Video player without color management is as good as dead.

**Comment by hooke007** [comment]
mpv never use Mac's ColorSync.

**Reply by sunlit777** [reply]
which means that iina is not usable since 90% of modern macs use wide gamut displays and and using non color managed player to view videos on such a display will result in unwatchable wrong colors
